### ISSUE TITLE
Cannot use scullog inside offline environment

### ISSUE THREAD

**Issue by OneB1t** [sanketbajoria/scullog#9]
Hello,
i tested your software for our internal development to ease developers access into container logs. But i have small issue that your software depends on third-party .js libraries which requires internet connection in order to work

Is is possible that you will release modified version for this type of application?

Thanks

**Comment by sanketbajoria** [comment]
Yeah.. it can be done.. i was also thinking to make it offline.. Another tool i have created cloudconnect, which is using scullog as dependency, which require it offline. I will release the offline version on main branch only by today or tomorrow.
